Task1/aga/IsPrime/IsPrime.sln

```diff
@@ -0,0 +1,22 @@
+﻿
```

Comment:
General remark - this commit contains binary files and some IDE generated content. Such things should never be commited and pushed to repo as it creates mess and risk of solution being dependent on local config.

1. Review content of each commit and commit only stuff that needs to be commited.
2. Learn how .gitignore works to make creating clean commites easier. https://git-scm.com/book/en/v2